File: app/src/main/java/com/sd/dialogview/handler/AppDialogProgressViewHandler.java
package com.sd.dialogview.handler;

import com.sd.lib.dialogview.DialogProgressView;
import com.sd.lib.dialogview.core.handler.IDialogProgressViewHandler;

public class AppDialogProgressViewHandler implements IDialogProgressViewHandler
{
    @Override
    public void setTextMsg(DialogProgressView dialogView, String text)
    {

    }

    @Override
    public int getContentViewResId(DialogProgressView dialogView)
    {
        return 0;
    }

    @Override
    public void onContentViewChanged(DialogProgressView dialogView)
    {

    }

    @Override
    public void onAttachedToWindow(DialogProgressView dialogView)
    {
        dialogView.setTextMsg("hook msg");
    }

    @Override
    public void onDetachedFromWindow(DialogProgressView dialogView)
    {

    }
}

File: app/src/main/java/com/sd/dialogview/handler/AppDialogProgressViewHandler.kt
package com.sd.dialogview.handler

import com.sd.lib.dialogview.DialogProgressView
import com.sd.lib.dialogview.core.handler.IDialogProgressViewHandler

class AppDialogProgressViewHandler : IDialogProgressViewHandler {
    override fun setTextMsg(dialogView: DialogProgressView, text: String?) {

    }

    override fun getContentViewResId(dialogView: DialogProgressView): Int {
        return 0
    }

    override fun onContentViewChanged(dialogView: DialogProgressView) {

    }

    override fun onAttachedToWindow(dialogView: DialogProgressView) {
        dialogView.setTextMsg("hook msg")
    }

    override fun onDetachedFromWindow(dialogView: DialogProgressView) {

    }
}